pennylane/templates/state_preparations/mottonen.py

```diff
@@ -266,7 +266,7 @@ def MottonenStatePreparation(state_vector, wires):
         norm = np.sum(np.abs(state_vector_values) ** 2)
     else:
         norm = np.sum(np.abs(state_vector) ** 2)
-    if not np.isclose(norm, 1.0, atol=1e-3):
+    if not math.isclose(norm, 1.0, abs_tol=1e-3):
```

Comment:
Big tolerance!

Reply:
Yes, I wonder why? Is this because the state could be a result from a previous stochastic algorithm? Maybe we should sneak in a tolerance change here if it does not cause tests to fail?